### ISSUE TITLE
Add support for policy based actions on CVSS score

### ISSUE THREAD

**Issue by nurmi** [anchore/anchore-engine#164]
Feature request is based on availability of CVSS information (see feature requests in #160 , #163 ).  Once CVSS information is made available in the data records, extend the vulnerabilities policy gate to include a trigger that is raised based on CVSS score information i addition to the existing 'Severity' trigger.

**Comment by ykosaraju** [comment]
+1 this would be nice

**Comment by SyCode7** [comment]
+1 this will be great similar to what I did [here ](https://github.com/maineffort/SecurityGateway/blob/5eadbff4ea5762b1bee3477dd63bdce43aa8f583/src/main/java/de/cavas/PolicyCheck.java) using OWASP ZAP for ensuring application security.

**Comment by nurmi** [comment]
We've been working on adding CVSS information and ability to use it for control/gate decisions in anchore engine (see also #163), and I thought it would be good to share the status of the functional prototype that we are preparing.  

Here, I've included an example illustrating new policy bundle parameters that can be used in the existing 'vulnerabilities' gate.  In this example, the gate will trigger iff a vulnerability is discovered and has an available and known CVSSV3 score.  Additional rules can be added to match vulnerabilities that do not have a CVSS score at all (omit the score checks altogether, or add a score check '= -1.0' for any/all score types).  Finally, note that the existing 'severity' parameter may also be still be supplied, to construct more nuanced checks.  The additional of these parameters does not alter backward compatbility, if the checks are omitted and severity is used as before, the behavior is unchanged.

```
                {
                    "action": "STOP", 
                    "gate": "vulnerabilities", 
                    "params": [
                        {
                            "name": "package_type", 
                            "value": "all"
                        }, 
                        {
                            "name": "cvssV3_baseScore", 
                            "value": "0.0"
                        },
                        {
                            "name": "cvssV3_baseScore_comparison", 
                            "value": ">="
                        },
                        {
                            "name": "cvssV3_exploitabilityScore", 
                            "value": "0.0"
                        }, 
                        {
                            "name": "cvssV3_exploitabilityScore_comparison", 
                            "value": ">="
                        },
                        {
                            "name": "cvssV3_impactScore", 
                            "value": "0.0"
                        }, 
                        {
                            "name": "cvssV3_impactScore_comparison", 
                            "value": ">="
                        }
...
...
...
}
```

**Comment by zhill** [comment]
Addressed in the v0.5.0 release
